packages/unity-bootstrap-theme/stories/molecules/tabbed-panels/tabbed-panels.js

```diff
@@ -1,94 +1,128 @@
-import { googleAnalytics } from '../../../src/js/googleAnalytics';
+import { googleAnalytics } from "../../../src/js/googleAnalytics";
 
 export const initTabs = function () {
-  'use strict';
-  jQuery(function () {
-    $(document).on('click', function (e) {
-      setButtonsCompatibility(e);
-    });
+  "use strict";
 
-    document.querySelectorAll('.uds-tabbed-panels').forEach((item) => {
-      const nav = item.querySelector('.nav-tabs');
-      nav.addEventListener('scroll', (event) => {
-        const scrollPos = event.target.scrollLeft;
-        const prevButton = item.querySelector('.scroll-control-prev');
-        const nextButton = item.querySelector('.scroll-control-next');
-        const atFarRight = nav.offsetWidth + scrollPos + 3 >= nav.scrollWidth;
-        prevButton.style.display = scrollPos === 0 ? 'none' : 'block';
-        nextButton.style.display = atFarRight ? 'none' : 'block';
-      });
-    });
+  // helpers functions
+  const setButtonsCompatibility = e => {
+    const targets = ["a", "button"];
+    if (targets.includes(e.target.localName)) {
+      e.target.focus();
+    }
+  };
 
-    $('.scroll-control-next').on('click', function (e) {
-      if (window.innerWidth > 992) {
-        slideNav(this, e, -1);
-      }
-    });
+  const slideNav = (clicked, e, direction) => {
+    e.preventDefault();
+    const parentNav = Array.from(clicked.parentElement.children).filter(child =>
+      child.classList.contains("nav-tabs")
+    );
 
-    $('.scroll-control-prev').on('click', function (e) {
-      if (window.innerWidth > 992) {
-        slideNav(this, e, 1);
-      }
-    });
+    let scrollPosition = parseInt(parentNav[0].dataset.scrollPosition, 10);
+
+    const navItems = Array.from(parentNav[0].querySelectorAll(".nav-item"));
 
-    $('.uds-tabbed-panels .scroll-control-prev').hide();
+    // get left value to interact with scroll
+    const rawLeftValue = getComputedStyle(parentNav[0]).left;
+    const sanitizedLeftValue = rawLeftValue.replace("px", "");
+    let scrollOffset = parseInt(sanitizedLeftValue, 10);
 
-    if ($('#nav-tab')[0].scrollWidth <= $('.uds-tabbed-panels').width()) {
-      $('.uds-tabbed-panels .scroll-control-next').hide();
+    if (direction == 1 && scrollPosition > 0) {
+      scrollPosition -= 1;
     }
-  });
+    if (scrollPosition < navItems.length - 1 && direction == -1) {
+      scrollPosition += 1;
+    }
+    parentNav[0].dataset.scrollPosition = scrollPosition;
 
-  const setButtonsCompatibility = (e) => {
-    const targets = ['a', 'button'];
-    if (targets.includes(e.target.localName)) {
-      e.target.focus();
+    scrollOffset = 0;
+    for (var i = 0; i < scrollPosition; i++) {
+      scrollOffset +=
+        navItems[i].offsetWidth +
+        parseInt(getComputedStyle(navItems[i]).marginLeft, 10) +
```

Comment:
This is more of a question, than a suggestion, I thought I saw BS5 preferring marginStart and marginEnd vs left and right.
Should we start using marginStart, marginEnd?

Reply:
I don't know. Could you share the documentation or where you read that?